BitwardenShared/UI/Vault/Vault/VaultList/VaultListProcessor.swift

```diff
@@ -86,7 +86,7 @@ final class VaultListProcessor: StateProcessor<
         case .refreshAccountProfiles:
             await refreshProfileState()
         case .refreshVault:
-            await refreshVault(isManualRefresh: true)
+            await refreshVault(isManualRefresh: false)
```

Comment:
🤔 IMO we should change the name of the parameter `isManualRefresh` to something else as in this case would be contradictory given that the user is actually manually refreshing the vault. I would use `forceSync` as in the end is what internally is being treated as in the repositories.
Same applies to Send processor and Vault+Send repositories.

Reply:
Good call.